reana_job_controller/schemas.py

```diff
@@ -41,6 +41,7 @@ class JobRequest(Schema):
     kerberos = fields.Bool(required=False)
     voms_proxy = fields.Bool(required=False)
     kubernetes_uid = fields.Int(required=False)
+    kubernetes_memory_limit = fields.Str(required=False)
```

Comment:
Dumping my thoughts, not for now: We should probably move all validation to this Marshmallow Schema class e.g:
- Compute backend (now in a view)
- Kubernetes memory limit, both for [regexp](https://marshmallow.readthedocs.io/en/stable/marshmallow.validate.html#marshmallow.validate.Regexp), and maximum value (now in the KubernetesJobManager)
- ....

Reply:
100% in to use Marshmallow wiser.

However, it's a bit tricky because the memory limit should only be validated when `job_manager_cls` is `KubernetesJobManager`, as it's not relevant for the rest of compute backends. And the schema contains mixed-up info (e.g. htcondor vs kubernetes params).

In other words, if I accidentally have `kubernetes_memory_limit` in my HTCondor workflow spec, should it be validated?